mkdocs.yml

```diff
@@ -46,6 +38,7 @@ theme:
     - "navigation.instant"
     - "navigation.tracking"
     - "navigation.top"
+    - "navigation.indexes"
```

Comment:
Regarding the material documentation this feature flag doesn't work in combination with `toc.integrate`
https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#integrated-table-of-contents

Reply:
Yeah but the strange thing is, it works :laughing: at least local testing. If you check out this branch locally you can see test it.

Reply:
Hehe cool, so let‘s try it 😉